src/cms-content/tooltips/index.ts

```diff
@@ -48,3 +51,34 @@ export const locationPageHeaderTooltipContent: Tooltip = getTooltipContentById(
   'location-page-header',
   infoToolTipContent.tooltip,
 );
+
+/* CCVI tooltips: */
+
+interface CcviSubscoreTooltip {
+  subscoreName: string;
+  id: string;
+  content: Markdown;
+}
+
+function getCcviTooltipById(id: string) {
+  const tooltipContent = ccviTooltipContent.subscoreTooltips.find(
+    item => item.id === id,
+  );
+  assert(
+    tooltipContent,
+    `Ccvi tooltip content unexpectedly not found for id: ${id}`,
+  );
+  return tooltipContent;
+}
```

Comment:
It might be easier to create a map instead of finding through the array each time:

```
const tooltipContentById = _.keyBy(ccviTooltipContent, item => item.id);
function getCcviTooltipById(id: string) {
  const tooltipContent = tooltipContentById[id];
  assert(
    tooltipContent,
    `Ccvi tooltip content unexpectedly not found for id: ${id}`,
  );
  return tooltipContent;
}
```